src/swap/saga.test.ts

```diff
@@ -608,29 +722,159 @@ describe(swapSubmitPreparedSaga, () => {
+          inAmount: {
+            value: BigNumber('0.0102'), // guaranteedPrice * sellAmount
+            tokenId: mockEthTokenId,
+          },
+          outAmount: {
+            value: BigNumber('0.01'),
+            tokenId: mockUSDCTokenId,
+          },
+          transactionHash: mockSwapTxReceipt.transactionHash,
+        })
+      )
+      .put(
+        transactionConfirmed('id-swap/saga-Swap/Execute', {
+          transactionHash: mockSwapTxReceipt.transactionHash,
+          block: mockSwapTxReceipt.blockNumber.toString(),
+          status: true,
+        })
+      )
+      .call([publicClient.ethereum, 'waitForTransactionReceipt'], { hash: '0x2' })
+      .run()
+    expect(mockViemWallet.signTransaction).toHaveBeenCalledTimes(2)
+    expect(mockViemWallet.sendRawTransaction).toHaveBeenCalledTimes(2)
+    expect(loggerErrorSpy).not.toHaveBeenCalled()
+
+    expect(ValoraAnalytics.track).toHaveBeenCalledTimes(1)
+    expect(ValoraAnalytics.track).toHaveBeenCalledWith(SwapEvents.swap_execute_success, {
+      toToken: '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
+      toTokenId: mockEthTokenId,
+      toTokenNetworkId: NetworkId['ethereum-sepolia'],
+      fromToken: mockUSDCAddress,
+      fromTokenId: mockUSDCTokenId,
+      fromTokenNetworkId: NetworkId['ethereum-sepolia'],
+      amount: '10000000000000000',
+      amountType: 'buyAmount',
+      price: '1',
+      allowanceTarget: '0xdef1c0ded9bec7f1a1670819833240f027b25eff',
+      estimatedPriceImpact: '0.1',
+      provider: '0x',
+      fromTokenBalance: '10000000000000000000',
+      swapApproveTxId: 'id-swap/saga-Swap/Approve',
+      swapExecuteTxId: 'id-swap/saga-Swap/Execute',
+      quoteToUserConfirmsSwapElapsedTimeInMs: 2_500,
+      quoteToTransactionElapsedTimeInMs: 10_000,
+      estimatedBuyTokenUsdValue: 0.005,
+      estimatedSellTokenUsdValue: 0.01,
+      web3Library: 'viem',
+      gas: 1384480,
+      maxGasFee: 0.01661376,
+      maxGasFeeUsd: 0.00830688,
+      gasUsed: 423252,
+      gasFee: 0.00211626,
+      gasFeeUsd: 0.00105813,
+      feeCurrency: undefined,
+      feeCurrencySymbol: 'ETH',
+      txCount: 2,
+      approveTxCumulativeGasUsed: 3_129_217,
+      approveTxEffectiveGasPrice: 5_000_000_000,
+      approveTxFeeCurrency: undefined,
+      approveTxFeeCurrencySymbol: 'ETH',
+      approveTxGas: 59_480,
+      approveTxMaxGasFee: 0.00071376,
+      approveTxMaxGasFeeUsd: 0.00035688,
+      approveTxGasUsed: 51_578,
+      approveTxGasFee: 0.00025789,
+      approveTxGasFeeUsd: 0.000128945,
+      approveTxHash: '0x1',
+      swapTxCumulativeGasUsed: 3_899_547,
+      swapTxEffectiveGasPrice: 5_000_000_000,
+      swapTxFeeCurrency: undefined,
+      swapTxFeeCurrencySymbol: 'ETH',
+      swapTxGas: 1_325_000,
+      swapTxMaxGasFee: 0.0159,
+      swapTxMaxGasFeeUsd: 0.00795,
+      swapTxGasUsed: 371_674,
+      swapTxGasFee: 0.00185837,
+      swapTxGasFeeUsd: 0.000929185,
+      swapTxHash: '0x2',
```

Comment:
holy heck, this is a long list 😅 do we need both this list and the assertions below?

Reply:
Yes just to be sure the math checks out and we don't base our analytics on wrong numbers.

This test is really a duplicate of the Celo one above, so I went ahead and used `it.each` to make it more succint.